core/execution/liquidity_provision.go

```diff
@@ -658,6 +658,7 @@ func (m *Market) cancelLiquidityProvision(
 			return err
 		}
 		m.broker.Send(events.NewLedgerMovements(ctx, []*types.LedgerMovement{tresp}))
+		m.collateral.RemoveBondAccount(party, m.GetID(), asset)
```

Comment:
you don't even check the error for it...

Reply:
Changed to panic now as you recommended